docs/docs/user-guide/permissions.md

```diff
@@ -0,0 +1,108 @@
+
+When a new object (database, schema, or table) is created, the owner of the object is the collaborator who created it. This automatic ownership assignment allows the creator to have immediate and full control over the objects they create, including the ability to manage permissions and transfer ownership if needed.
+
+### Database Permissions
+
+Database permissions control access and actions at the database level.
+
+- **Owner**: Each database has an owner who has full control over the database, including managing permissions and transferring ownership.
+- **Granted Access**: Specific permissions can be granted to roles for various actions within the database.
+- **Transfer Ownership**: The current owner can transfer ownership to another role, granting them full administrative control.
+
+For each database, the following permission levels can be granted:
+
+- **Connect**: Allows the role to access and connect to the database.
+- **Create**: Includes Connect permissions and allows the role to create new schemas within the database.
+- **Custom**: Enables the granular setting of permissions beyond the predefined options.
+
+### Schema Permissions
+
+Schema permissions control access and actions at the schema level.
+
+- **Owner**: Each schema has an owner who has full control over the schema, including managing permissions and transferring ownership.
+- **Granted Access**: Specific permissions can be granted to roles for various actions within the schema.
+- **Transfer Ownership**: The current owner can transfer ownership to another role, granting them full administrative control over the schema.
+
+For each schema, the following permission levels can be granted:
+
+- **Read**: Allows the role to access the schema and view its objects.
+- **Create**: Includes Read permissions and allows the role to create new tables within the schema.
+- **Custom**: Enables the granular setting of permissions beyond the predefined options.
+
+### Table Permissions
+
+Table permissions control access and actions at the table level.
+
+- **Owner**: Each table has an owner who has full control over the table, including managing permissions, transferring ownership, and modifying the table's structure (such as adding, removing, or altering columns).
+- **Granted Access**: Specific permissions can be granted to roles for various actions on the table.
+- **Transfer Ownership**: The current owner can transfer ownership to another role, granting them full administrative control over the table.
+
+For each table, the following permission levels can be granted:
+
+- **Read**: Allows the role to access the table and read records.
+- **Write**: Includes Read permissions and allows the role to insert, update, and delete records in the table.
+- **Custom**: Enables the granular setting of permissions beyond the predefined options.
```

Comment:
We could include a link to the Postgres docs regarding the possible custom privileges.

Reply:
I've added a link to: https://www.postgresql.org/docs/current/ddl-priv.html